tabbycat/utils/tables.py

```diff
@@ -906,7 +919,8 @@ def add_debate_results_columns(self, debates):
                 team = debate.get_team(side)
 
                 subtext = None if (all_sides_confirmed or not debate.sides_confirmed) else side_abbrs[side]
-                cell = self._team_cell(team, show_emoji=False, subtext=subtext)
+                cell = self._team_cell(team, show_emoji=False, subtext=subtext, show_iron=True,
+                    iron=debateteam.iron, iron_prev=debateteam.iron_prev)
```

Comment:
The `iron` and `iron_prev` attributes aren't guaranteed to exist here, because there's no guarantee that it was called with `debate_set_with_prefetches(iron=True)`. The attributes are called even if not used, because they're evaluated before being passed to `self._team_cell`. It might be tempting to guarantee that the attribute exists, but because there's nothing about _any_ `DebateTeam` that can guarantee that it exists (and because this is not a core function of `DebateTeam`, and because you don't know who will call `add_debate_results_columns()` in the future), it's better for this code not to expect it to exist. For example, we could use `getattr(debateteam, 'iron', 0)`.

Sadly, this isn't just hypothetical—the adjudicator record and public results pages now both crash with `AttributeError: 'DebateTeam' object has no attribute 'iron'`.

Reply:
On a different note, since this annotation is only used in one place (here), it might make more sense to add to `cell` from this function, rather than pass more arguments to `self._team_cell()`? (This part is up for discussion.)